pkg/mcs/tso/server/server.go

```diff
@@ -71,7 +71,7 @@ func (s *Server) GetHTTPClient() *http.Client {
 // CreateServerWrapper encapsulates the configuration/log/metrics initialization and create the server
 func CreateServerWrapper(args []string) (context.Context, context.CancelFunc, bs.Server) {
 	cfg := tso.NewConfig()
-	err := cfg.Parse(os.Args[1:])
+	err := cfg.Parse(args)
```

Comment:
Why do we change this?

Reply:
This is a bug introduced last time. This code currently is dead code and new change is on the way.

Reply:
Changed back for now. will address it in separate pr for service initialization.